### PR TITLE
Feature/issue 43/treat plt files

### LINKED ISSUE #43: Plt module
## Abstract
introduce Plt module that treat dialyzer plt file.

```ocaml
type t
val from_file : filename -> (t, exn) Result.t
```

## Related link

- dialyzer_plt.erl
- abstract_term_format

### PR DESCRIPTION
**Pull request by yoshihiro503**
fix #43 

introduce parsing plt function

```ocaml
val of_file : filename -> (Plt.t, exn) Result.t
```

### CODE REVIEW

File: lib/erl_type.ml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+open Base
```
Comment (amutake) [inline lib/erl_type.ml]: Is `erl_type.mli` missing?

File: lib/etf_util.ml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+open Base
```
Comment (amutake) [inline lib/etf_util.ml]: Is `etf_util.mli` missing?

File: lib/erl_type.ml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+-opaque erl_type() :: ?any | ?none | ?unit | #c{}.
+v}
+@see <https://github.com/erlang/otp/blob/OTP-21.1.1/lib/hipe/cerl/erl_types.erl>
+ *)
+type t =
+  | Any
+  | None
+  | Unit
+  | Atom of string list
```
Comment (amutake) [inline lib/erl_type.ml]: memo: `string list` seems strange for me, but in `erl_types.erl`, `atom` has a set
https://github.com/erlang/otp/blob/OTP-21.1.1/lib/hipe/cerl/erl_types.erl#L325

File: test/unit-test/test_plt.ml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+module E = Etf_util
+
+let%expect_test "Plt.of_file" =
+  let print filename =
+    Plt.of_file filename
+    |> [%sexp_of: (Plt.t, exn) Result.t]
+    |> Expect_test_helpers_kernel.print_s
+  in
+
+  print "plt/test01.plt";
```
Comment (amutake) [inline test/unit-test/test_plt.ml]: Could you write how to generate `test01.plt` as a comment?

Comment (yoshihiro503) [reply test/unit-test/test_plt.ml]: Wrote

Comment (amutake) [reply test/unit-test/test_plt.ml]: Thank you!

**Pull request merged**